src/slic3r/GUI/Plater.cpp

```diff
@@ -5250,8 +5250,10 @@ void Plater::add_model(bool imperial_units/* = false*/)
         return;
 
     std::vector<fs::path> paths;
-    for (const auto &file : input_files)
+    for (const auto &file : input_files) {
         paths.emplace_back(into_path(file));
+        wxGetApp().mainframe->add_to_recent_projects(file);
```

Comment:
If you're going to change the menu name, shouldn't all the related functions etc. be refactored to reflect such change?

Reply:
Probably yes, I just would like make changes as local as possible, according to guidelines. But if code maintainers are agree, I certainly can rename methods and variables.